components/Alert/alert.css

```diff
@@ -0,0 +1,43 @@
+.alert {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: 1rem;
+  border-radius: 5px;
+}
+
+.alert-error {
+  background-color: #ffe4e6;
```

Comment:
instead of using hex values directly, you should use CSS variables

Reply:
> The primary colour which you have used has very less contrast with white, so you can take a darker shade if possible

Reply:
Thanks Shraddha for pointing it out, but since I was not planning to use these colors in any other file that's why I didn't declare it as variable. I have updated these colors now.